lighthouse-core/scripts/i18n/bake-ctc-to-lhl.js

```diff
@@ -119,15 +118,13 @@ function saveLhlStrings(path, localeStrings) {
  * @param {string} dir
  * @return {Array<string>}
  */
-function collectAndBakeCtcStrings(dir) {
+export function collectAndBakeCtcStrings(dir) {
```

Comment:
small change here... can use `dir` instead of `LH_ROOT`, and `relativePath` was not necessary.